lua/neural.lua

```diff
@@ -228,6 +228,4 @@ function Neural.query(prompt, type, args)
     end
 end
 
-Neural.setup()
```

Comment:
The real issue at hand is that this was caused by the setup being performed twice.

Reply:
Yeah that's the issue, good catch :+1: